app/views/admin/accounting/transactions/_transactions_grid_definition.html.slim

```diff
@@ -1,6 +1,10 @@
 = define_grid(@transactions_grid, hide_csv_button: true,
     html: {class: 'rowlink table-hover', data: {link: 'row'}}) do |g|
 
+  - if action_name == 'show'
+    - g.row_attributes do |transaction|
+      - {onclick: "document.location = '#{admin_loan_transaction_path(loan_id: @loan.try(:id), id: transaction)}';"}
```

Comment:
It would be better to use a `link_to` here.

For example, this is from the loans grid.
```
= define_grid(@loans_grid, show_filters: :show, hide_csv_button: true,
    html: {class: 'rowlink table-hover', data: {link: 'row'}}) do |g|

    - g.blank_slate { @loans_grid.no_records_at_all = true; "" }

    - g.column name: t('common.id'), attribute: 'id', filter: true, in_csv: false, detach_with_id: :id_filter do |loan|
      - link_to(loan.id, admin_loan_path(loan.id))
```

Reply:
gotcha

Will update this

Reply:
yea, I remember why I couldn't use the link_to helper

the row_attributes expects a hash to be returned so the helper does not work with it

Reply:
Ok. Don't worry about it then. Thanks.